SampleProject/ViewControllers/VOKTableViewController.m

```diff
@@ -0,0 +1,38 @@
+//
+//  VOKTableViewController.m
+//  Vokoder Sample Project
+//
+//  Copyright © 2015 Vokal.
+//
+
+#import "VOKTableViewController.h"
+#import "UIViewController+VOKConvenience.h"
+#import "VOKPersonDataSource.h"
+
+@interface VOKTableViewController ()
+
+@property (strong, nonatomic) VOKPersonDataSource *dataSource;
+
+@end
+
+@implementation VOKTableViewController
+
+- (void)loadView
+{
+    [super loadView];
+    [self layoutNavBarButtons];
+    [self setupCustomMapper];    
+    [self setupDataSource];
```

Comment:
There's no nib or storyboard for their view controllers so I think it's about the same either way.